nload/plan.sh

```diff
@@ -0,0 +1,41 @@
+pkg_name=nload
+pkg_origin=core
+pkg_version="0.7.4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_source="https://github.com/rolandriegel/${pkg_name}/archive/v${pkg_version}.tar.gz"
```

Comment:
Please add a `pkg_description` and `pkg_upstream_url`.  https://travis-ci.org/habitat-sh/core-plans/builds/464177154#L536